examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/ConfigureFragment.kt

```diff
@@ -85,6 +89,18 @@ class ConfigureFragment : Fragment() {
         dataStoreUtils.saveSdkConfig(SdkConfiguration(apiKey, proxyUrl, useAmazonStore))
     }
 
+    private fun setupSupportedStoresRadioButtons() {
```

Comment:
Honestly, many things here belong to view models... But this was the quick and dirty approach which I think it's fine for now.